src/spurt/graph/_interface.py

```diff
@@ -25,16 +38,26 @@ def npoints(self) -> int:
 
     @property
     def points(self) -> np.ndarray:
-        """Return vertices as (npts, ndim)."""
+        """Return points/ vertices/ nodes as (npts, 2)."""
 
     @property
-    def simplices(self) -> np.ndarray | list[list[int]]:
-        """Return simplices as (nsimplex, ...)."""
+    def cycles(self) -> np.ndarray | list[list[int]]:
+        """Return cycles/ simplices as (nsimplex, ...)."""
 
     @property
-    def boundary(self) -> np.ndarray | list[list[int]]:
-        """Return list of arcs forming outer boundary of graph."""
+    def links(self) -> np.ndarray:
+        """Return list of links/ arcs in the graph as (nlinks, 2)."""
+
+
+@runtime_checkable
+class PlanarGraphInterface(GraphInterface, Protocol):
+    """
+    Interface to a 2D planar graph.
+
+    This provides an additional access to the boundary (convex_hull) of the
+    graph.
+    """
 
     @property
-    def links(self) -> np.ndarray:
-        """Return list of links in the graph as (nlinks, 2)."""
+    def boundary(self) -> np.ndarray | list[list[int]]:
+        """Return list of arcs forming outer boundary of graph."""
```

Comment:
`DelaunayGraph.boundary` is returning the convex hull of the `scipy.spatial.Delaunay` object.

https://github.com/isce-framework/spurt/blob/ba110bd52f0fd2b3a84683ce44e52b0dfe79dfb1/src/spurt/graph/_delaunay.py#L63-L65

The docstring for [`scipy.spatial.Delaunay.convex_hull`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.Delaunay.convex_hull.html#scipy.spatial.Delaunay.convex_hull) says that it's a set of *vertices* -- not a set of *arcs*.

>Vertices of facets forming the convex hull of the point set.

So that sounds like it's supposed to be an Nx2 array of (x,y) coordinates of each boundary vertex. Is that right?

Reply:
Have added more documentation. It is vertice indices. After implementing #6, I realized that this is more a utility function and doesn't really get used in the processing